docs/cli/plugin_implementation_guide.md

```diff
@@ -49,6 +49,17 @@ To add the admin repository use `tanzu plugin repo add -n admin -b tanzu-cli-adm
 
 To add the builder plugin use `tanzu plugin install builder`
 
+#### Installing Admin Plugins
+
+With the [context-aware plugin discovery](/docs/design/context-aware-plugin-discovery-design.md) been enabled within Tanzu CLI admin plugins can be installed from local source as mentioned below:
+
+1. Download the admin plugin tarball or zip file from the release artifacts and extract it
+1. Run `tanzu plugin lsit --local /path/to/admin-plugins/linux-amd64-admin` to list available admin plugins
+1. Run `tanzu plugin install all --local /path/to/admin-plugins/linux-amd64-admin` command to install all admin plugins. User can user `plugin-name` instead of `all` to install specific plugin
```

Comment:
```suggestion
1. Run `tanzu plugin install all --local /path/to/admin-plugins/linux-amd64-admin` command to install all admin plugins. User can use `_plugin-name_` instead of `all` to install a specific plugin.
```